instaclustr/resource_cluster.go

```diff
@@ -592,6 +592,45 @@ func resourceCluster() *schema.Resource {
 					return
 				},
 			},
+			"kafka_connect_credential": {
+				Type:	  schema.TypeList,
```

Comment:
Why a list with one element? As opposed to a set or something like that?

Reply:
Arjun gave the idea of the list with one element and that worked easily. I tried the TypeSet before and couldn't make it to work yet, must be some syntax stuff and I couldn't seem to find examples easily. Would there be a strong preference one over the other?

Reply:
Not particularly, just curious